lua/crates/toml.lua

```diff
@@ -855,6 +865,14 @@ function M.parse_crates(buf)
             if crate then
                 crate.section = dep_section
                 table.insert(crates, Crate.new(crate))
+            else
+                local name_s, name, name_e = line:match [[^%s*()([^%s]+)()%s*$]]
+                if name_s and name and name_e then
+                    working_crate = {
```

Comment:
Similarly here, if there are multiple locations with an unfinished assignment, only the last one can be completed.